app/server/models/transfer_account.py

```diff
@@ -212,30 +207,61 @@ def make_initial_disbursement(self, initial_balance=None):
         return disbursement
 
     def initialise_withdrawal(self, withdrawal_amount):
-        from server.utils.credit_transfers import make_withdrawal_transfer
+        from server.utils.credit_transfer import make_withdrawal_transfer
         withdrawal = make_withdrawal_transfer(withdrawal_amount,
                                               send_account=self,
                                               automatically_resolve_complete=False)
         return withdrawal
 
+    def _bind_to_organisation(self, organisation):
+        if not self.organisation:
+            self.organisation = organisation
+        if not self.token:
+            self.token = organisation.token
 
     def __init__(self,
                  blockchain_address: Optional[str]=None,
-                 private_key: Optional[str]=None,
-                 organisation: Optional[Organisation]=None,
+                 bind_to_entity: Optional[Union[Organisation, User]]=None,
                  account_type: Optional[TransferAccountType]=None,
+                 private_key: Optional[str] = None,
                  **kwargs):
 
         super(TransferAccount, self).__init__(**kwargs)
```

Comment:
when trying to get seed data to work, I had a tonnnne of trouble with inconsistent bindings between an entity controlling a transfer_account and the transfer_account itself, so decided to try and unify the process a little